js/index.js

```diff
@@ -1,7 +1,96 @@
+console.log("I'm ready!");
+
 // Iteration 1: Names and Input
+let hacker1 = "Pip";
+console.log(`"The driver's name is ${hacker1}"`);
 
+let hacker2 = "Pip";
+console.log(`"The navigator's name is ${hacker2}"`);
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+  console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+  } else if (hacker2.length > hacker1.length) {
+  console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+  } else {
+  console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);
+}
+
+// Iteration 3: Loops
+let driverName = "";
+let navName = "";
+
+for (let i = 0; i < hacker1.length; i++) {
+  driverName = driverName + hacker1[i].toUpperCase() + " ";
+}
+
+for (let i = hacker2.length - 1; i >= 0; i--) {
+  navName += hacker2[i];
+}
+
+if (hacker1.localeCompare(hacker2) === 1) {
```

Comment:
Just to improve legibility, sometimes it is good to see if it is easier using a switch. 💅🏼
`switch (hacker1.localeCompare(hacker2)) {
  case 1:
     console.log("The driver's name goes first. ")
    break;
  case 0:
     console.log("What?! You both have the same name?")
     break;
  case -1: 
    console.log("Yo, the navigator goes first definitely.")
    break;
}`

Reply:
Aah makes sense! Thanks for the tip! 😃